### PR TITLE
Unignore Tests

### PR DESCRIPTION
**Pull request by fppt**

### CODE REVIEW

**Comment by fppt** [comment]
@aelred don't merge this until I ask you to review please. I am going to be incrementally unignoring tests with this PR.

**Comment by fppt** [comment]
@aelred this can be reviewed and merged

File: grakn-test/src/test/java/ai/grakn/test/graql/query/InsertQueryTest.java [hunk 1]
```diff
@@ -329,9 +328,20 @@ public void testIterateMatchInsertResults() {
...
+        AskQuery query456 = qb.match(var().isa("language").has("name", "456").has("name", "HELLO")).ask();
+
+        //Check if one of the matches have had the insert executed correctly
+        boolean oneExists;
+        if(query123.execute()){
+            oneExists = !query456.execute();
+        } else {
+            oneExists = query456.execute();
+        }
+        assertTrue("A match insert was not executed correctly for only one match", oneExists);
```
Comment (aelred) [inline grakn-test/src/test/java/ai/grakn/test/graql/query/InsertQueryTest.java]: Beautiful time to use the XOR operator:
`boolean oneExists = query123.execute() ^ query456.execute();`

Comment (fppt) [inline grakn-test/src/test/java/ai/grakn/test/graql/query/InsertQueryTest.java]: Oooo you right.

**Pull request merged**